synapse/lib/storm.py

```diff
@@ -2351,7 +2351,9 @@ async def _snapFromOpts(self, opts):
                 if view is None:
                     raise s_exc.NoSuchView(mesg=f'No such view iden={viewiden}', iden=viewiden)
 
-                self.user.confirm(('view', 'read'), gateiden=viewiden)
+                if not self.asroot:
```

Comment:
Could this just be `self.confirm()` since this is a runtime?

Reply:
Yeah that works too